app/src/main/java/com/example/supercookie/ConfigHelper.java

```diff
@@ -7,23 +7,23 @@
 
 public class ConfigHelper {
 
-  public static final String SQUARE_LOCATION_ID_FOR_GOOGLE_PAY = "REPLACE_ME";
+  public static final String GOOGLE_PAY_MERCHANT_ID = "REPLACE_ME";
   private static final String CHARGE_SERVER_HOST = "REPLACE_ME";
   private static final String CHARGE_SERVER_URL = "https://" + CHARGE_SERVER_HOST + "/";
 
   public static boolean serverHostSet() {
     return !CHARGE_SERVER_HOST.equals("REPLACE_ME");
   }
 
-  public static boolean squareLocationIdSet() {
-    return !SQUARE_LOCATION_ID_FOR_GOOGLE_PAY.equals("REPLACE_ME");
+  public static boolean merchantIdSet() {
+    return !GOOGLE_PAY_MERCHANT_ID.equals("REPLACE_ME");
   }
 
   public static void printCurlCommand(String nonce) {
     String uuid = UUID.randomUUID().toString();
     Log.d("ExampleApplication",
         "Run this curl command to charge the nonce:\n"
-            + "curl --request POST https://connect.squareup.com/v2/locations/SQUARE_LOCATION_ID_FOR_GOOGLE_PAY/transactions \\\n"
+            + "curl --request POST https://connect.squareupsandbox.com/v2/payments \\\n"
```

Comment:
nit: `squareupsandbox` -> `squareup`

Reply:
Actually, we explicitly want to target the Sandbox URL for this, so the first time running this would actually not cause a production charge.